packages/openactive-integration-tests/test/features/payment/prepayment-unavailable/implemented/opportunity-paid-incomplete-payment-details-test.js

```diff
@@ -0,0 +1,20 @@
+const { FeatureHelper } = require('../../../../helpers/feature-helper');
+const { errorTests } = require('../../common');
+
+const testOpportunityCriteria = 'TestOpportunityBookableNonFreePrepaymentUnavailable';
+const controlOpportunityCriteria = 'TestOpportunityBookableFree';
+const expectedPrepayment = 'https://openactive.io/Unavailable';
+const expectedError = 'UnnecessaryPaymentDetailsError';
+const bReqTemplateRef = 'incorrectOrderDueToMissingIdentifierInPaymentProperty';
+
+FeatureHelper.describeFeature(module, {
+  testCategory: 'payment',
+  testFeature: 'prepayment-unavailable',
+  testFeatureImplemented: true,
+  testIdentifier: 'opportunity-paid-incomplete-payment-details',
+  testName: 'IncompletePaymentDetailsError must be returned in the case that payment details are not supplied',
+  testDescription: 'An unsuccessful end to end booking, because identifier is missing in the payment property in B request.',
+  testOpportunityCriteria,
+  controlOpportunityCriteria,
+},
+errorTests(expectedPrepayment, expectedError, bReqTemplateRef));
```

Comment:
I don't think this test makes any sense. It attempts, when prepayment is unavailable, to make a payment, but with a missing `identifier`. The payment property itself is erroneous, whether or not it's missing an identifier. Because that case is covered by packages/openactive-integration-tests/test/features/payment/prepayment-unavailable/implemented/opportunity-paid-unnecessary-payment-error-test.js, I think this test can safely be deleted

Reply:
I deleted it

Reply:
Ah, so this test is supposed to test to ensure that if the identifier is omitted an `UnnecessaryPaymentDetailsError` is returned, instead of a `IncompletePaymentDetailsError`. The latter could be returned when the identifier is missing, even though the whole payment object is unnecessary, if the booking implementation under test has its error handling logic the wrong way around).

Noting the metadata was erroneously inconsistent here with the test itself.

Reply:
Okay, I'll add this test in a separate PR and update the metadata.